src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -94,7 +121,7 @@ public InfoboxBuilderPage() {
   public InfoboxBuilderPage openNew(String templateName) {
```

Comment:
The purpose of that is to test whole flow with it's entry point. It's navigating to template page and selecting the Infobox template type. By modifying the URL, we would test it without that important part.